server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.iotdb.db.integration;
+
+import com.clearspring.analytics.util.Lists;
+import org.apache.iotdb.db.utils.EnvironmentUtils;
+import org.apache.iotdb.jdbc.Config;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author yuqi
+ * @mail yuqi5@xiaomi.com
+ * @description your description
+ * @time 27/9/20 22:56
+ **/
+public class IoTDBTestResultSet {
+    private static List<String> sqls = Lists.newArrayList();
+    private static Connection connection;
+
```

Comment:
```suggestion
public class IoTDBTestResultSet {
  private static List<String> sqls = Lists.newArrayList();
  private static Connection connection;

```
Actually, we use double space symbols to intend the code.

Reply:
Right